docs/AccessControl.md

```diff
@@ -113,6 +113,246 @@ This specifies that the file is expected to be transmitted from the caller to
+visible, otherwise the application might rather report authorization
+errors.
+
+
+### SRT_REJX_BAD_MODE
+
+The mode specified in the `m` key in StreamID is not supported for this request.
+This is in case of having read-only or write-only resources, as well as the
+interactive (bidirectional) access is not predicted as a valid access to this
+resource.
+
+
+### SRT_REJX_UNACCEPTABLE
+
+The requested parameters specified in SocketID cannot be satisfied for the
+requested resource. Also when m=publish and the data format is not acceptable.
+This is a general error reporting the unsupported data format for the data
+in case when it appears to be wrong when sending, or a restriction on the
+data format (as specified in the details of the resource specification) cannot
+be provided when receiving.
+
+
+### SRT_REJX_CONFLICT
+
+The resource being accessed is already locked for modification. This error
+should only be reported for `m=publish` in case when the resource being
+accessed is read-only. Note that it doesn't mean that the resource is not
+modifiable under current authorization restrictions, but that it is currently
+disabled completely for modifications.
+
+
+### SRT_REJX_NOTSUP_MEDIA
+
+The media type is not supported by the application. The media type is
+specified in the `t` key, although the currently standard types are
+`stream`, `file` and `auth`, an application may extend this list, and
+also it's not obliged to support all of the standard ones.
+
+
+### SRT_REJX_LOCKED
+
+The resource being accessed is locked for any access. This is similar to
+`SRT_REJX_CONFLICT`, but in this case it's locked for reading and writing.
+This is for a case when the resource should be shown as existing and in
+general available for the client, just temporarily blocked for access at
+the moment.
+
+
+### SRT_REJX_FAILED_DEPEND 
+
+The dependent entity for the request is not present. In this case the
+dependent entity is the session, which should be specified in the `s`
+key. This means that the specified session ID is nonexistent or it
+has already expired.
+
+
+### SRT_REJX_ISE
+
+Internal server error. This is for a general case when the request has
+been correctly verified and no problems related to this have been found,
+the processing of the request has started and the activity was about to
+be correctly performed, just there was some unexpected error during this
+process.
+
+
+### SRT_REJX_UNIMPLEMENTED
+
+The request was correctly recognized, but the current version doesn't
+support it. This should be reported for a case, when some features to
+be specified in the StreamID request are supposed to be supported in a
+predictable feature, but the current version of the server does not
+support it, or the support for this feature in this version has been
+temporarily blocked. This shouldn't be reported in case of deprecated
+features or features that were decided as no longer supported.
```

Comment:
What is the difference between "deprecated features" and
"features that were decided as no longer supported"?

Reply:
"deprecated" is such that the request for it is tolerated as a kinda backward compatibility (specifying them is not an error), but they may generate a warning, or it's announced in the documentation that this feature is about to be removed in future versions. "features that were decided as no longer supported" are those that are simply not supported (and trying to use them results in an error), although they were supported in the earlier version, or more precisely, in the previous major version they were deprecated.

Reply:
Suggested edit: "This shouldn't be reported for existing features that are being deprecated, or older features that are no longer supported.